spec/index.html

```diff
@@ -733,125 +733,139 @@ <h3>Literals</h3>
       <a data-cite="I18N-GLOSSARY#dfn-case-sensitive">case sensitive matching</a>
-      (see description of string comparison in 
-      <a href="#rdf-strings" class="sectionRef"></a>)
-      except for language tags, where the comparison is performed using 
+      (see description of string comparison in
+      <a href="#rdf-strings" class="sectionRef"></a>).
+      Comparison of the [=language tags=] is performed using
       <a data-cite="I18N-GLOSSARY#dfn-case-sensitive">ASCII case-insensitive matching</a>.
-      Thus, two literals can have the same value
-      without being the same <a>RDF term</a>.
-      For example:</p>
-
-    <pre>
-      "1"^^xs:integer
-      "01"^^xs:integer
-    </pre>
-
-    <p>denote the same <a data-lt="literal value">value</a>, but are not the
-      same literal <a>RDF terms</a> and are not
-      <a>term-equal</a> because their
-      <a>lexical forms</a> differ.</p>
+    </p>
+
+    <section>
+      <h2>Representation of literals</h2>
+
+      <p>Some concrete syntaxes MAY support
+        <dfn data-lt="simple literal" class="export">simple literals</dfn> consisting of only a
+        <a>lexical form</a> without any <a>datatype IRI</a>, <a>language tag</a>, or <a>base direction</a>.
+        Simple literals are syntactic sugar for abstract syntax
+        <a>literals</a>
+        with the <a>datatype IRI</a>
+        <code>http://www.w3.org/2001/XMLSchema#string</code>
+        (which is commonly abbreviated as <code>xsd:string</code>).
+      </p>
+
+      <p>
+        Similarly, most concrete syntaxes represent
+        <a>language-tagged strings</a> and <a>directional language-tagged strings</a> without
+        the <a>datatype IRI</a> because it always equals either
+        <code>http://www.w3.org/1999/02/22-rdf-syntax-ns#langString</code> (<code>rdf:langString</code>)
+        or <code>http://www.w3.org/1999/02/22-rdf-syntax-ns#dirLangString</code> (<code>rdf:dirLangString</code>), respectively.
+      </p>
+
+      <p>
+        Any [=string=] complying with [[!BCP47]] MAY be used to represent a [=language tag=] in concrete syntaxes or implementation.
+        Such strings MAY be case normalized,
+        (for example, by canonicalizing as defined by
+        <a data-cite="bcp47#section-4.5">BCP 47 section 4.5</a>).
+        On the contrary, an implementation MAY preserve the case from the original representation,
+        provided that it processes it in a case-insensitive manner.
+      </p>
+
+      <aside class=note>
+        The treatment of language tags has changed between RDF 1.1 and RDF 1.2.
+        In RDF 1.1, `"chat"@fr` and `"chat"@FR` were representing two distinct terms, but implementations had license to replace one with the other (which most did).
+        In RDF 1.2, they are now representing the exact same literal, i.e., the case difference in the concrete syntax does not propagate into the abstract syntax.
+      </aside>
```

Comment:
I think this is too strong. RDF 1.1 was careful to allow implementations to normalize (typically lower-case) the language tag when parsing, so differences in case may not result in different terms and triples could collapse.

Perhaps just say it was a point of ambiguity but 1.2 behavior generally represents what implementations actually do.

Reply:
Proposal:
* replace "were representing two distinct terms" with "were *theoretically* representing two distinct terms"
* explain a little more than "replacing" in RDF 1.1 uses some form of normalization
* add a sentence explicitly stating that most implementations will not be affected, because they normalize already

```suggestion
      <aside class=note>
        The treatment of language tags has changed between RDF 1.1 and RDF 1.2.
        In RDF 1.1, `"chat"@fr` and `"chat"@FR` were theoretically representing two distinct terms, but implementations had license to replace one with the other via some form of normalization.
        In RDF 1.2, they are now representing the exact same literal, i.e., the case difference in the concrete syntax does not propagate into the abstract syntax.
        Since most RDF 1.1 implementation do normalize language tags internally, they will not be impacted by this change?
      </aside>
```

Reply:
For the last of the new sentences that you propose: This is a claim. I wouldn't make claims here. So, better: Hence, RDF 1.1 implementations that normalize language tags internally will not be impacted by this change.

If you want to keep your sentence, however, then: "implementation" -> "implementations", and no question mark at the end.

Otherwise, this change looks good to me.

Reply:
Thanks @hartig . I propose to change "most" into "many", which is closer to what we can actually claim. 

But I can back this claim with data. Out of the 10 implementations I have tested (Comunica, Corese, Dydra, GraphDB, Jena, Oxigraph, Python RDFlib, Qlever, Ruby RDF, Virtuoso), only Virtuoso considers that `"chat"@fr` and `"chat"@FR` are not the same term (but, ironically, displays them both as `"chat"@fr` :smile:).

```suggestion
      <aside class=note>
        The treatment of language tags has changed between RDF 1.1 and RDF 1.2.
        In RDF 1.1, `"chat"@fr` and `"chat"@FR` were theoretically representing two distinct terms, but implementations had license to replace one with the other via some form of normalization.
        In RDF 1.2, they are now representing the exact same literal, i.e., the case difference in the concrete syntax does not propagate into the abstract syntax.
        Since many RDF 1.1 implementations do normalize language tags internally, they will not be impacted by this change.
      </aside>
```

Reply:
Looks good!